doc/release-notes/release-notes-3.1.0-rc1.md

```diff
@@ -51,7 +51,7 @@ Alfredo Garcia (28):
       add estimatedheight to getblockchaininfo
       add documentation and command line parsing to afl scripts
       get fuzzing options from directory
-      add bool argument to get balance in satoshis to rpc getreceivedbyaddress
+      add bool argument to get balance in zatoshis to rpc getreceivedbyaddress
```

Comment:
Revert release notes changes (so that it's easy to search commits by these messages).